From cf0f72ecd2c0466544fd937b01a536a81b5cc914 Mon Sep 17 00:00:00 2001
From: Timon Sassor <timon.sassor@governikus.de>
Date: Thu, 18 Sep 2025 15:23:44 +0200
Subject: a11y iOS: Provide translations for the new Rotor names for TextEdit

VoiceOver has the Rotor that allows it to change the text navigation on
editable components between character, word, line, sentence etc..
We already extended the iOS a11y bridge and provide the translations
for the user facing-strings here.

[ChangeLog][Accessibility][iOS] Provide translations for the new
custom Rotor names for TextEdit

Fixes: QTBUG-139943
Change-Id: Idba7c540939638f34bda5115bc273008aa98afad
---
 translations/qtbase_de.ts | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git x/qttranslations/translations/qtbase_de.ts y/qttranslations/translations/qtbase_de.ts
index 48e999acdad9dab7737b1d582164531db9393eb7..479ccc84f86d39b65f055d28a431626970a18681 100644
--- x/qttranslations/translations/qtbase_de.ts
+++ y/qttranslations/translations/qtbase_de.ts
@@ -7564,6 +7564,18 @@ Die minimal erforderliche Version von Direct2D ist %1 Die auf diesem System inst
         <source>unchecked</source>
         <translation>nicht markiert</translation>
     </message>
+        <message>
+        <source>Character by character</source>
+        <translation>Zeichenweise</translation>
+    </message>
+    <message>
+        <source>Word by word</source>
+        <translation>Wortweise</translation>
+    </message>
+    <message>
+        <source>Sentence by sentence</source>
+        <translation>Satzweise</translation>
+    </message>
 </context>
 <context>
     <name>QFactoryLoader</name>
